peps/pep-0396.rst

```diff
@@ -3,13 +3,22 @@ Title: Module Version Numbers
 Version: $Revision: 65628 $
 Last-Modified: $Date: 2008-08-10 09:59:20 -0400 (Sun, 10 Aug 2008) $
 Author: Barry Warsaw <barry@python.org>
-Status: Rejected
+Status: Withdrawn
```

Comment:
We need confirmation from @warsaw before this is merged (I don't think the rejection was ever formalised, hence the lack of a resolution link).

Reply:
Direct link to the rendered PEP preview: https://pep-previews--4074.org.readthedocs.build/pep-0396/